pkgs/cupertino_http/lib/src/cupertino_api.dart

```diff
@@ -344,6 +344,32 @@ class URLSessionConfiguration
   bool get discretionary => _nsObject.discretionary;
   set discretionary(bool value) => _nsObject.discretionary = value;
 
+  /// Additional headers to send with each request.
+  ///
+  /// Note that the getter for this field returns a **copy** of the headers.
+  ///
+  /// See [NSURLSessionConfiguration.HTTPAdditionalHeaders](https://developer.apple.com/documentation/foundation/nsurlsessionconfiguration/1411532-httpadditionalheaders)
+  Map<String, String>? get httpAdditionalHeaders {
+    if (_nsObject.HTTPAdditionalHeaders == null) {
+      return null;
+    }
+    final headers = ncb.NSDictionary.castFrom(_nsObject.HTTPAdditionalHeaders!);
+    return stringDictToMap(headers);
```

Comment:
Consider avoiding the `!` with an if-case and local non-null variable.

```suggestion
    if (_nsObject.HTTPAdditionalHeaders case var additionalHeaders?) {
      final headers = ncb.NSDictionary.castFrom(additionalHeaders);
      return stringDictToMap(headers);
    }
    return null;
```